pyproject.toml

```diff
@@ -35,6 +35,8 @@ liccheck = "^0.3.12"
 bandit = "^1.6"
 black = {version = "^18.3-alpha.0", allow-prereleases = true}
 flake8 = "^3.7"
+python-freeipa = "^1.0"
+Faker = "^4.0.0"
```

Comment:
Whenever you change the `pyproject.toml` file, you have to run `poetry lock` afterwards so it updates the lock file. Or you can use `poetry add -D Faker` instead of editing the file, it will do both.